src/pages/Dashboard/Dashboard.jsx

```diff
@@ -1,23 +1,29 @@
 import "./dashboard.css";
 
 const Dashboard = () => (
-    <div className="content__card">
-        <div className="content__resume">
-          <h3 className="content__label">Ingresos</h3>
-          <span className="content__value">$1200000</span>
-        </div>
-        <div className="content__resume">
-          <h3  className="content__label">Egresos</h3>
-          <span className="content__value">$4000000</span>
-        </div>
-        <div className="content__resume">
-          <h3  className="content__label">Clientes</h3>
-          <span className="content__value">5</span>
-        </div>
-        <div className="content__resume">
-          <h3  className="content__label">Proximas Citas</h3>
-          <span className="content__value">10</span>
-        </div>
+  <div className="dashboard">
+    <h3 className="dashboard__subtitle">Dashboard</h3>
+    <h1 className="dashboard__title">Bienvenid@s a Iconik</h1>
```

Comment:
this should be a div and not a h1, because it is not a title, also change the className